### PR TITLE
move anakinra to pharmaceuticals

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

Replaces #605 
Due to the restructure + not having the necessary write permissions on other fork, I think this is the easiest way to get a version of #605 that can be incorporated. There was great feedback on #605 so we should keep an eye on it!

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by agitter** [comment]
Do we want to work on merging #605 with this new pull request or close it?  It felt very preliminary to me.

**Comment by rando2** [comment]
@agitter I was going to try to provide feedback on the content from #605 but ran into some really bad merge conflict issues! I was planning to give my feedback here so that it can be more easily incorporated.

File: content/20.pharmaceuticals.md [hunk 1]
```diff
@@ -651,6 +651,19 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+###### Anakinra
+**Anticipated Mechanism.**
```
Comment (rando2) [inline content/20.pharmaceuticals.md:657]:
```suggestion

```
We changed the format to less explicitly use these subheadings, so I'll remove them in the text here!

File: content/20.pharmaceuticals.md [hunk 2]
```diff
@@ -651,6 +651,20 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+Anakinra is a recombinant of IL-1 receptor antagonist (IL-1Ra) that blocks binding of IL-1α, or IL-1β to IL-1R1.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:658]:
```suggestion
Anakinra is a recombinant of the IL-1 receptor antagonist (IL-1Ra) and blocks the binding of IL-1α and IL-1β to IL-1R1.
```

File: content/20.pharmaceuticals.md [hunk 3]
```diff
@@ -651,6 +653,20 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+##### Anakinra
+**Anticipated Mechanism.**
+Anakinra is a recombinant of IL-1 receptor antagonist (IL-1Ra) that blocks binding of IL-1α, or IL-1β to IL-1R1.
+Its main task is to reduce inflammation caused by IL-1-mediation, which finds a broad spectrum of applications, including acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
```
Comment (rando2) [inline content/20.pharmaceuticals.md:659]:
```suggestion
It is primarily used to reduce inflammation by mediating IL-1, providing a broad spectrum of applications that include the treatment of acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
```

File: content/20.pharmaceuticals.md [hunk 4]
```diff
@@ -651,6 +653,20 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+The application of Anakinra features a very safety profile, and has a very low rate of side effects and prolonged duration.
+
+**Current Evidence.**
```
Comment (rando2) [inline content/20.pharmaceuticals.md:662]:
```suggestion
```

File: content/20.pharmaceuticals.md [hunk 5]
```diff
@@ -651,6 +653,20 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+
+In two retrospective cohort studies, first at the San Raffaele Hospital in Milan, Italy with 29 patients treated with high dosages of anakinra and 16 controls [@doi:10.1016/S2665-9913(20)30127-2], and second at Groupe Hospitalier Paris Saint-Joseph Paris, France, with 96 participants in France, among which 52 were treated with the drug and 44 were administered standard protocols, anakinra has proven to progressively improve respiratory function, dicrease likelihood of application of invasive mechanical ventilation, and significantly reduce mortality [@doi:10.1016/S2665-9913(20)30164-8].
+
+**Summary.**
```
Comment (rando2) [inline content/20.pharmaceuticals.md:666]:
```suggestion
```

File: content/20.pharmaceuticals.md [hunk 6]
```diff
@@ -651,6 +653,20 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+##### Anakinra
+**Anticipated Mechanism.**
+Anakinra is a recombinant of IL-1 receptor antagonist (IL-1Ra) that blocks binding of IL-1α, or IL-1β to IL-1R1.
+Its main task is to reduce inflammation caused by IL-1-mediation, which finds a broad spectrum of applications, including acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
+The application of Anakinra features a very safety profile, and has a very low rate of side effects and prolonged duration.
```
Comment (rando2) [inline content/20.pharmaceuticals.md:660]: Wow, it's quite hard to find good safety profiles of Anakinra! I will keep looking.

**Review by rando2 — COMMENTED** [review]
@athril thank you for starting all of this! I was curious, is there more content you want to add, or should I work on fleshing it out to complement the Tocilizumab section above?

**Comment by athril** [comment]
Hi @rando2!

It's been a while since I wrote this section and I think it would be
worthwhile to revisit and update it.
Is there any specific deadline for changes?

Thank you!

**Review by RLordan — COMMENTED** [review]
Hey @rando2 and @athril, I gave a quick review. Feel free to add, edit, or dismiss the suggestions. Here is another paper I came across that was promising but not as significant - https://doi.org/10.1186/s13054-020-03364-w

There is certainly a few reviews out there registering their interest for anakinra.

**Comment by rando2** [comment]
Hey @athril, Thank you so much! If there's any chance to get some updates by the end of next week, I think that should work for me to give you feedback! Otherwise, we could just add a mention to anakinra as an alternative possible interleukin inhibitor and flesh it out once more data is a available!

**Comment by athril** [comment]
I have found three other studies. The first one is from Dec 10, 2020:  https://doi.org/10.1186/s13054-020-03364-w
The second one (retrospective) even suggests reduced mortality with anakinra for patients not responding to  corticosteroids or corticosteroids+tocilizumab: https://doi.org/10.1007/s11739-020-02600-z
Another paper in Lancet (what a hit, from today!) is probably the first randomized trial. They claim anakinra didn't improve the outcome of mild to moderate covid19 pneumonia patients: https://www.thelancet.com/journals/lanres/article/PIIS2213-2600(20)30556-7/fulltext Will be creating the link to the patients shortly and extending the description.

**Review by athril — COMMENTED** [review]
I have added a couple of lines of text. Thank you @RLordan for your comments, I have just noticed that we have found the same papers ;)

File: content/20.pharmaceuticals.md [hunk 7]
```diff
@@ -651,6 +653,25 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
 Additionally, different studies used different dosages, number of doses, and methods of administration; ongoing research may be needed to optimize administration of TCZ [@doi:10.1016/j.jcv.2020.104443], although similar results were reported by one study for intravenous and subcutaneous administration [@doi:10/d2pk].
 Clinical trials that are in progress are likely to provide additional insight into the effectiveness of this drug for the treatment of COVID-19 along with how it should be administered.
 
+##### Anakinra
+Anakinra is a recombinant of the IL-1 receptor antagonist (IL-1Ra) and blocks the binding of IL-1α and IL-1β to IL-1R1.
+It is primarily used to reduce inflammation by mediating IL-1, providing a broad spectrum of applications that include the treatment of acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
```
Comment (rando2) [inline content/20.pharmaceuticals.md:658]:
```suggestion
Its primary function is to reduce inflammation by mediating IL-1, providing a broad spectrum of applications that include the treatment of acute stroke [@doi10.1136/jnnp.2004.054882], rheumatoid arthritis [@doi:10.3899/jrheum.090074], and several hereditary, systemic or local inflammatory diseases [@doi:10.1056/NEJMoa055137, @doi:10.1186/s12969-017-0147-3, @doi:10.3389/fphar.2018.01157].
```
@RLordan's suggestion that got lost while I was trying to figure out how to merge in all the suggestions!

File: content/20.pharmaceuticals.md [hunk 8]
```diff
@@ -651,6 +653,26 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+Anakinra has a favorable safety profile with very low rates of reported side effects in therapeutic studies for rheumatoid arthritis [@doi:10.1136/ard.2005.048371; @doi:10.1093/rheumatology/kew208; @url:https://www.globenewswire.com/news-release/2020/12/22/2149573/0/en/FDA-Approves-KINERET-anakinra-for-the-Treatment-of-Deficiency-of-IL-1-Receptor-Antagonist-DIRA.html].
+
+To date, a couple of cohort studies have been conducted to determine the therapeutic value of anakinra. 
+The first study at the San Raffaele Hospital in Milan, Italy, intravenously treated 29 patients with high doses of anakinra versus 16 controls, all of whom received the standard of care at the time [@doi:10.1016/S2665-9913(20)30127-2]. 
+A second cohort study at the Groupe Hospitalier Paris Saint-Joseph in Paris, France, subcutaneously administered anakinra to 52 patients who also received standard care at the time versus 44 historical control subjects. 
+In both studies, anakinra seemed to improve respiratory function, decrease the likelihood of requiring invasive mechanical ventilation, and in Paris, it significantly reduced mortality [@doi:10.1016/S2665-9913(20)30164-8].
```
Comment (agitter) [inline content/20.pharmaceuticals.md:665]:
```suggestion
In both studies, anakinra seemed to improve respiratory function, decrease the likelihood of requiring invasive mechanical ventilation, and in Paris, it significantly reduced mortality [@doi:10/gg4smj]. 
```

Can we be more precise?  "Seemed to improve" sounds qualitative, whereas other trial summaries have been more quantitative.

File: content/20.pharmaceuticals.md [hunk 9]
```diff
@@ -651,6 +653,26 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+
+To date, a couple of cohort studies have been conducted to determine the therapeutic value of anakinra. 
+The first study at the San Raffaele Hospital in Milan, Italy, intravenously treated 29 patients with high doses of anakinra versus 16 controls, all of whom received the standard of care at the time [@doi:10.1016/S2665-9913(20)30127-2]. 
+A second cohort study at the Groupe Hospitalier Paris Saint-Joseph in Paris, France, subcutaneously administered anakinra to 52 patients who also received standard care at the time versus 44 historical control subjects. 
+In both studies, anakinra seemed to improve respiratory function, decrease the likelihood of requiring invasive mechanical ventilation, and in Paris, it significantly reduced mortality [@doi:10.1016/S2665-9913(20)30164-8]. 
+Similar observation was made in [@doi:10.1007/s11739-020-02600-z], where anakinra was used as a second tier after no response to standard corticosteroids or combination of corticosteroids and tocilizumab treatments.
```
Comment (agitter) [inline content/20.pharmaceuticals.md:666]:
```suggestion
A similar observation was made in [@doi:10.1007/s11739-020-02600-z], where anakinra was used as a second tier after no response to standard corticosteroids or combination of corticosteroids and tocilizumab treatments.
```

What was the result of this trial?

File: content/20.pharmaceuticals.md [hunk 10]
```diff
@@ -651,6 +653,26 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+
+In another retrospective cohort study carried by Radboud University Medical Center (Nijmegen, The Netherlands) and Bernhoven Hospital (Uden, The Netherlands), Anakinra was found to effectively reduce hyperinflammation clinical signs for critically ill patients.[@doi:10.1186/s13054-020-03364-w].
```
Comment (agitter) [inline content/20.pharmaceuticals.md:668]:
```suggestion
In another retrospective cohort study carried by Radboud University Medical Center (Nijmegen, The Netherlands) and Bernhoven Hospital (Uden, The Netherlands), Anakinra was found to effectively reduce hyperinflammation clinical signs for critically ill patients [@doi:10.1186/s13054-020-03364-w]. 
```

As above, I recommend being more quantitative about what we mean by "effectively reduce".

File: content/20.pharmaceuticals.md [hunk 11]
```diff
@@ -651,6 +653,26 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+A second cohort study at the Groupe Hospitalier Paris Saint-Joseph in Paris, France, subcutaneously administered anakinra to 52 patients who also received standard care at the time versus 44 historical control subjects. 
+In both studies, anakinra seemed to improve respiratory function, decrease the likelihood of requiring invasive mechanical ventilation, and in Paris, it significantly reduced mortality [@doi:10.1016/S2665-9913(20)30164-8]. 
+Similar observation was made in [@doi:10.1007/s11739-020-02600-z], where anakinra was used as a second tier after no response to standard corticosteroids or combination of corticosteroids and tocilizumab treatments.
+
+In another retrospective cohort study carried by Radboud University Medical Center (Nijmegen, The Netherlands) and Bernhoven Hospital (Uden, The Netherlands), Anakinra was found to effectively reduce hyperinflammation clinical signs for critically ill patients.[@doi:10.1186/s13054-020-03364-w]. 
+
+The previous findings were not confirmed in a recent randomized clinical trial carried in 16 hospitals in France, which included 116 patients, with 59 in anakinra cohort and 57 in control group.
```
Comment (agitter) [inline content/20.pharmaceuticals.md:670]:
```suggestion
The previous findings were not confirmed in a recent randomized clinical trial carried in 16 hospitals in France, which included 116 patients, with 59 in the anakinra cohort and 57 in the control group. 
```

File: content/20.pharmaceuticals.md [hunk 12]
```diff
@@ -651,6 +653,26 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+In both studies, anakinra seemed to improve respiratory function, decrease the likelihood of requiring invasive mechanical ventilation, and in Paris, it significantly reduced mortality [@doi:10.1016/S2665-9913(20)30164-8]. 
+Similar observation was made in [@doi:10.1007/s11739-020-02600-z], where anakinra was used as a second tier after no response to standard corticosteroids or combination of corticosteroids and tocilizumab treatments.
+
+In another retrospective cohort study carried by Radboud University Medical Center (Nijmegen, The Netherlands) and Bernhoven Hospital (Uden, The Netherlands), Anakinra was found to effectively reduce hyperinflammation clinical signs for critically ill patients.[@doi:10.1186/s13054-020-03364-w]. 
+
+The previous findings were not confirmed in a recent randomized clinical trial carried in 16 hospitals in France, which included 116 patients, with 59 in anakinra cohort and 57 in control group. 
+In this study anakinra was not confirmed to reduce mortality for COVID-19 and mild-to-moderate pneumonia patients treated intravenously.
```
Comment (agitter) [inline content/20.pharmaceuticals.md:671]: Same request, can you please add more about the trial results or statistics?

File: content/20.pharmaceuticals.md [hunk 13]
```diff
@@ -651,6 +653,26 @@ Interest in TCZ as a treatment for COVID-19 was supported by two meta-analyses t
...
+The limitations of this study are a different treatment protocol than the previous studies (half of the dosage, which was admitted intravenously) and not analyzing severe COVID-19 cases [@10.1016/S2213-2600(20)30556-7].
+
+Anakinra may be a promising treatment option worthy of further investigation for patients with severe forms of COVID-19.
```
Comment (agitter) [inline content/20.pharmaceuticals.md:674]: This conclusion seems too strong or optimistic to me given the evidence above.

**Review by agitter — CHANGES_REQUESTED** [review]

**Review by rando2 — COMMENTED** [review]
This is just some preliminary formatting feedback. I think it would be good to include a bit of additional info about these studies (namely, what was SOC? what were the primary and secondary outcomes? what were the statistical differences between the two groups?) 

I'm happy to help with this but have a few other things I need to wrap up tonight, so I won't be able to contribute until tomorrow!

**Comment by rando2** [comment]
Hey @athril, I wasn't sure if you needed help fleshing this out or responding to mine and @agitter's feedback? We are getting close to needing to send the paper out to authors. I can definitely just add a sentence to note that this drug in the same category as TCZ is under consideration, if it would be easier to expand on it moving forward!

**Comment by athril** [comment]
Hi! I will take a look at it tomorrow. Thank you!

**Comment by agitter** [comment]
If we're still interested in anakinra, we can create an issue for this new trial: https://doi.org/10.7554/eLife.66125

**Comment by RLordan** [comment]
> If we're still interested in anakinra, we can create an issue for this new trial: https://doi.org/10.7554/eLife.66125

@athril led on this one

**Review by athril — COMMENTED** [review]
> If we're still interested in anakinra, we can create an issue for this new trial: https://doi.org/10.7554/eLife.66125

Thanks for sharing!
This trial doesn't change much in my opinion. Anakinra remains an interesting lead, but the number of participants is too small to make a substantial difference.

**Comment by rando2** [comment]
Closing this PR due to the new paper structure

**Pull request closed without merging**